openshift/template.yaml

```
@@ -133,7 +139,11 @@ parameters:
   value: "false"
 - name: JOBS_AUTO_REPORT_SCHEDULE
   value: "@weekly"
-- name: JOBS_AUTO_REPORT_WEBHOOK_SECRET_KEY
-  value: webhook_url
-- name: JOBS_AUTO_REPORT_WEBHOOK_SECRET_NAME
-  value: jobs-auto-report-slack-webhook
+- name: JOBS_AUTO_REPORT_SLACK_BOT_TOKEN_NAME
+  value: jobs-auto-report-slack-bot-token
```

if the value is mandatory, don't set any value here. it's better to fail to deploy because the value is not set then failing to startup the pod because the default is wrong

---

can also set `required: true`

---

@danmanor yes, you can refer to the documentation: https://docs.openshift.com/container-platform/4.11/openshift_images/using-templates.html#templates-writing-parameters_using-templates

---

the values set are not placeholders